### DISCUSSION TITLE
Uploader request method

### DISCUSSION THREAD

**Discussion by abdullaev388** [rsuite/rsuite#2540] [Q&A] [answered]
Hello, i want to use uploader but it always uses POST method. How can i specify the request method for example i want to use PATCH

**Comment by simonguo** [comment, marked as answer]
Hi @abdullaev388 

Uploader will add a `method` prop.  https://github.com/rsuite/rsuite/pull/2541

```tsx
<Uploader method="PATCH" />
```

**Reply by abdullaev388** [reply]
@simonguo Thank you very much sir!
